src/Framework/Hooks/CommonConditions.cs

```diff
@@ -1,41 +1,46 @@
-                ["Seasons"] = (valueToCheck, _) => valueToCheck.Split(' ').Any(s => s == SDate.Now().Season),
-                ["DaysOfWeek"] = (valueToCheck, _) => valueToCheck.Split(' ').Any(
-                        d => d.ToLower() == SDate.Now().DayOfWeek.ToString().ToLower()),
-                ["Friendship"] = (valueToCheck, _) => CheckFriendshipCondition(valueToCheck),
-                ["MailReceived"] = (valueToCheck, _) => CheckReceivedMailCondition(valueToCheck),
-                ["EventSeen"] = (valueToCheck, _) => CheckEventSeenCondition(valueToCheck),
-                ["MinDaysPlayed"] = (valueToCheck, _) => Game1.Date.TotalDays >= Convert.ToInt32(valueToCheck),
-                ["MaxDaysPlayed"] = (valueToCheck, _) => Game1.Date.TotalDays <= Convert.ToInt32(valueToCheck),
-                ["DaysPlayed"] = (valueToCheck, _) => Game1.Date.TotalDays == Convert.ToInt32(valueToCheck),
-                ["IsPlayerMarried"] = (valueToCheck, _) => ParseBool(valueToCheck) == Game1.player.isMarried(),
-                ["QuestAcceptedInPeriod"] = (valueToCheck, managedQuest) => IsQuestAcceptedInPeriod(valueToCheck, managedQuest),
-                ["QuestAcceptedDate"] = (valueToCheck, managedQuest) => IsQuestAcceptedDate(valueToCheck, managedQuest),
-                ["KnownCraftingRecipe"] = (valueToCheck, _) => Game1.player.craftingRecipes.ContainsKey(valueToCheck),
-                ["KnownCookingRecipe"] = (valueToCheck, _) => Game1.player.cookingRecipes.ContainsKey(valueToCheck),
-                ["Random"] = (valueToCheck, _) => Game1.random.NextDouble() < Convert.ToDouble(valueToCheck) / 100, // Chance is in %
-            };
+using StardewModdingAPI.Utilities;
+using StardewValley;
+using StardewValley.Network;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace QuestFramework.Framework.Hooks
+{
+    internal class CommonConditions
+    {
+        internal protected static IMonitor Monitor => QuestFrameworkMod.Instance.Monitor;
+        public static Dictionary<string, Func<string, CustomQuest, bool>> GetConditions()
+        {
+            return new Dictionary<string, Func<string, CustomQuest, bool>>()
+            {
+                ["Weather"] = (valueToCheck, _) => valueToCheck.ToLower().GetCurrentWeatherName() == valueToCheck,
+                ["Date"] = (valueToCheck, _) => SDate.Now() == Utils.ParseDate(valueToCheck),
+                ["Days"] = (valueToCheck, _) => Utility.parseStringToIntArray(valueToCheck).Any(d => d == SDate.Now().Day),
+                ["Seasons"] = (valueToCheck, _) => valueToCheck.ToLower().Split(' ').Any(s => s == SDate.Now().Season),
+                ["DaysOfWeek"] = (valueToCheck, _) => valueToCheck.Split(' ').Any(
+                        d => d.ToLower() == SDate.Now().DayOfWeek.ToString().ToLower()),
+                ["FriendshipLevel"] = (valueToCheck, _) => CheckFriendshipLevel(valueToCheck), //Emily 7

```

Comment:
You are removed condition `Friendship` and broke API. Can you please make the old `Friendship` as alias to `FriendshipLevel` and if someone use the old print a warning to console `Friendship` is deprecated, use `FriendshipLevel` instead? Just warning, the old must work as worked before, but with warning in console.